### PR TITLE
Rename folder `ccc` to `src`

### PR DESCRIPTION
**Pull request by local-minimum**
Since we're getting more stuff that needs compiling and we want to reuse components and such this moves all the ccc-files and changes some names were needed.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
     entry: {
-        ccc: ['./scanomatic/ui_server_data/js/ccc/index.jsx'],
+        ccc: ['./scanomatic/ui_server_data/js/src/ccc_index.jsx'],
```
Comment (gdetrez) [inline webpack.config.js]: Maybe just `ccc.js`?

File: scanomatic/ui_server_data/js/src/api.js [hunk 2]
```diff
@@ -41,6 +39,8 @@ class API {
     }
 }
 
+export const HasJquery = () => !!$;
```
Comment (gdetrez) [inline scanomatic/ui_server_data/js/src/api.js]: Remove

Comment (local-minimum) [reply scanomatic/ui_server_data/js/src/api.js]: fixed

File: webpack.config.js (hunk 1 above)
Comment (local-minimum) [reply webpack.config.js]: fixed

**Pull request merged**